interfaces/portal/src/app/pages/program-payment-transaction-list/program-payment-transaction-list.page.ts

```diff
@@ -264,9 +295,7 @@ export class ProgramPaymentTransactionListPageComponent {
       return false;
     }
 
-    return this.transactionsResponse
-      .data()
-      .data.some((payment) => payment.status === TransactionStatusEnum.error);
+    return this.areThereFailedTransactions();
```

Comment:
aha, was still a needed refactor because of the pagination and it only looked at error transactions in the first page? Good catch.